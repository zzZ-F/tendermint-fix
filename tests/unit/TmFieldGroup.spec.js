import { mount } from "@vue/test-utils"
import htmlBeautify from "html-beautify"
import { TmFieldGroup } from "@/index"

describe("TmFieldGroup", () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(TmFieldGroup, {
      slots: { default: '<input class="fake-field" placeholder="Fake" />' }
    })
  })

  it("has the expected html structure", () => {
    expect(htmlBeautify(wrapper.html())).toMatchSnapshot()
  })

  it("has a slot with a field", () => {
    expect(wrapper.findAll(".fake-field").length).toBe(1)
  })
})
