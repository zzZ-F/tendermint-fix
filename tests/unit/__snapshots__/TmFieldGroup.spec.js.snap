// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmFieldGroup has the expected html structure 1`] = `
<div class="tm-field-group">
  <input placeholder="Fake" class="fake-field">
</div>
`;
